agents/wiregrid_kikusui/src/pmx.py

```diff
@@ -0,0 +1,341 @@
+# Built-in python modules
+import time as tm
+import serial as sr
+import sys as sy
+import os
+
+# Communication modules
+this_dir = os.path.dirname(__file__)
+sy.path.append(os.path.join(
+    this_dir, "..", "..","..", "MOXA"))
+import moxaSerial as mx  # noqa: E402
```

Comment:
I'm not sure this path is valid. It would probably be better to move code that looks like it's going to be shared between this Agent and the hwp rotation agent (#157) to the core socs library (i.e. under `socs/socs/`), rather than manipulating the path manually here.